src/gwt/src/org/rstudio/studio/client/workbench/WorkbenchList.java

```diff
@@ -34,9 +34,6 @@
    void remove(String item);
    void clear();
    
-   // status
-   boolean contains(String item);
```

Comment:
@jjallaire I had added this in the previous incantation of this PR, but it was misguided as one should be listening on ListChangedEvents to ensure they get an up-to-date MRU list.